working/3616 - enum value shorthand/proposal-simple-lrhn.md

```diff
@@ -266,11 +266,11 @@ like `.parse(input).abs()`, then the recognized expression, `.parse(input)`
 in this example, likely has no context type._
 
 Expressions of the forms <code>.new\<*typeArgs*\></code> or 
-<code>.new\<*typeArgs*\></code> (as a prefix of a `<staticMemberShorthand> <selector>*`
+<code>.new\<*typeArgs*\>(*args*)</code> (as a prefix of a `<staticMemberShorthand> <selector>*`
 production, or the entire chain) are compile-time errors, just like
 the corresponding <code>*T*.new\<*typeArgs*\></code>
-and <code>*T*.new\<*typeArgs*\></code> already are, whether used as instantiated
-tear-off or invoked.
+and <code>*T*.new\<*typeArgs*\>(*args*)</code> already are, whether used as
```

Comment:
I believe the original code is what @lrhn intended here, but I'm not sure.

Reply:
The original code looks like "Expressions of the forms `.new<typeArgs>` or `.new<typeArgs>`...just like the corresponding `T.new<typeArgs>` and `T.new<typeArgs>`...". 

`(T).new<typeArgs>` is repeated twice. I believe that the second one was intended to have an arguments part.